ting_search.module

```diff
@@ -973,5 +973,15 @@ function ting_search_submit($form, &$form_state) {
  */
 function _ting_search_quote($string) {
   $cqlDoc = new TingSearchCqlDoctor($string);
-  return $cqlDoc->string_to_cql($string);
+  // Hold cql query for further changes.
+  $return = $cqlDoc->string_to_cql();
+
+  // Issue 1392 filter on agency if required.
```

Comment:
I would prefer it if we did not refer to issues as the primary source of information about a block of code. The comment should at least describe what is going on here (if the code if not readable) and why. Then we can refer to external sources for additional information if needed.

In this case the commit message will also contain information about the source of the change.